MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/IEnggDiffractionView.h

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 #include <vector>
+#include <QStringList>
```

Comment:
This include should not be here (it adds Qt-dependency in a header that doesn't need it.
But this doesn't do any harm now. It can be fixed in the follow-up branch.